chips/msp432/src/lib.rs

```diff
@@ -0,0 +1,140 @@
+#![crate_name = "msp432"]
+#![crate_type = "rlib"]
+#![feature(asm, const_fn)]
+#![no_std]
+#![allow(unused_doc_comments)]
```

Comment:
Remove this `allow`, it shouldn't be needed (though it was apparently used elsewhere; #1992...)

Reply:
Ok!